src/tools/interop/idt/capture/ecosystem/play_services/command_map.py

```diff
@@ -0,0 +1,38 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+properties_and_commands = {
+    'android_model': 'shell getprop ro.product.model',
```

Comment:
It would be more efficient to call `shell getprop` then parse out the fields on interest?

Reply:
Yes will change this.

Reply:
Thanks, this did increase the efficiency for getprop; updated! 
dumpsys didn't seem worth doing this for. Streaming the whole output seemed to take too long...